scripting/get5_apistats.sp

```diff
@@ -196,10 +207,20 @@ static void CheckForLogo(const char[] logo) {
   }
 }
 
-static int LogoCallback(Handle request, bool failure, bool successful, EHTTPStatusCode status,
+static int LogoCallback(Handle request, bool failure, bool requestSuccessful, EHTTPStatusCode statusCode,
                         int data) {
-  if (failure || !successful) {
-    LogError("Logo request failed, status code = %d", status);
+  int status = view_as<int>(statusCode);
```

Comment:
Same as above. Break into two and don't read response or HTTP code if unsuccessful request.